ansible/playbook_Test_Scripts/Test_Script

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+printf "Enter the URL:  "			#User Input section
+read gitURL
+printf "Retain the VM? [y/n]\n" 
+read vmChoice
+
+
+cd ~/documents/					
+mkdir Unix_PB_Test_Folder
+						#Setup
+cd Unix_PB_Test_Folder
+mkdir Log_Files
+git clone $gitURL
+cd ${gitURL##*/}
+cd ansible
+
+for OS in Ubuntu1804 Ubuntu1604 CentOS6 	#For all tested OSs / Playbooks
+do
```

Comment:
Is this firing up 3 vagrants in that case?

Reply:
It fires up 3 vagrants, one after the other, and halts one (or destroys one) before starting the next